data/filedefs/filetypes.HCL.conf

```diff
@@ -0,0 +1,40 @@
+# For complete documentation of this file, please see Geany's main documentation
+[styling=C]
+# style "preprocessor" as comment, as for HCL lines starting with "#" are comments
+preprocessor=comment
+
+[keywords]
+# all items must be in one line
+primary=data local locals module provider resource terraform var
```

Comment:
There is neither `variable` nor `output`, yet those look like keywords in the `simple.ft` test file, aren't they?